scalding-serialization/src/main/scala/com/twitter/scalding/serialization/OrderedSerialization.scala

```diff
@@ -101,6 +101,30 @@ object OrderedSerialization {
     case NonFatal(e) => CompareFailure(e)
   }
 
+  def viaTransform[T, U](
+    packFn: T => U,
+    unpackFn: U => T)(implicit otherOrdSer: OrderedSerialization[U]): OrderedSerialization[T] =
+    new OrderedSerialization[T] {
```

Comment:
I'm not sure how we could use that feature right now? we could always add that in later though if we find a use